File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Height:
    int h;
    do
    {
        h = get_int("Height: ");
    }while (h < 1 || h > 8);

    int row = 0;
    while (row < h)
    {
        //Space 1:
        for (int i = 0; i < h - row - 1; i++)
        {
            printf(" ");
        }

        //Left:
        for (int j = 0; j < row + 1; j++)
        {
            printf("#");
        }

        //Space 2:
        printf("  ");

        //Right:
        for (int j = 0; j < row + 1; j++)
        {
            printf("#");
        }
        printf("\n");
        row++;
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int
while True:
    height = get_int("Height:")
    if height > 0 and height < 9:
        break



for row in range(height):
   # Space 1:
   for i in range(height - row - 1):
       print(" ", end="")

   # Left:
   for j in range(row + 1):
       print("#", end="")

   # Space 2:
   print("  ", end="")

   # Right:
   for j in range(row + 1):
       print("#", end="")

   print()